docs-src/references/dynamic-configuration.md

```diff
@@ -61,65 +61,47 @@ testGetMapPropertyKey:
 
 ### Constraints
 
-Some dynamic configuration keys are set globally, and others can be customized on a Namespace or Task Queue level.
+You can define constraints on some dynamic configuration keys to set specific values that apply on a Namespace or Task Queue level.
+Not defining constraints on a dynamic configuration key sets the values globally across the Cluster.
 
 - To set global values for the configuration key with no constraints, use:
 
   ```yaml
-  keyName:
-    - value: "my-value"
+  history.rps: # Rate limit on the number of requests made per second to each History service host.
+    - value: 5000 # The default value for this is 3000.
   ```
 
 - For keys that can be customized at Namespace level, you can specify multiple values for different Namespaces in addition to one default value that applies globally to all Namespaces.
   To set values at a Namespace level, use `namespace` (String) as shown in the following example.
 
   ```yaml
-  keyName:
+  frontend.rps: # Rate limit on the number of requests sent to the Frontend host.
     - constraints: {} # Sets default value that applies to all Namespaces
-      value: "value-for-all-the-rest"
+      value: 2000 # The defaul value for this key on the history host is 3000.
   - constraints: {namespace: "namespace1"} # Sets value for "namespace1" Namespace
-      value: "value-for-namespace1"
+      value: "4000"
```

Comment:
pretty sure this takes int values

Reply:
lol yeah! missed removing the quotes here. updated.